### DISCUSSION TITLE
ESP8266 - 4x Relay board

### DISCUSSION THREAD

**Discussion by DSRocker** [arendst/Tasmota#12004] [Q&A] [answered]
Hi,
I have 2 ESP8266 Relay boards with 4 Relay on each board. I flashed tasmota to the ESPs. It works fine. But when power on the board, the first ESP ist for mabe 0,5 seconds on and than off again.
I use the newest tasmota.
The Relays are on GPIO: 12, 13, 14, 16.

**Comment by ascillato** [comment, marked as answer]
That is a hardware limitation. That blink is how the Esp8266 boots and can not be controlled by software.
Some pins have several toggles or different startup level until the esp8266 starts the software in flash.

Please, check the ESP8266 datasheet for more information on that.

**Reply by sfromis** [reply]
With GPIO16 known to be high at boot, I'd say that it is a "rather questionable design" to use that for a relay output.

**Reply by DSRocker** [reply]
you are right... The first Relay is on GPIO16 :-/ I will fix this with my soldering iron ;-)

On the one board, the relay 1 (GPIO16) stays on after reboot...

**Reply by Jason2866** [reply]
Gpio 4 and Gpio 5 are a good choice

**Reply by DSRocker** [reply]
I soldered a wire from GPIO 4 to Relay. It works perfect! I bought this bourd in a onlineshop (china). Bad design but very cheap.
Thanks!
